comp/core/tagger/impl-remote/remote.go

```diff
@@ -109,23 +109,97 @@ type Options struct {
 
 // NewComponent returns a remote tagger
 func NewComponent(req Requires) (Provides, error) {
-	remoteTagger, err := newRemoteTagger(req.Params, req.Config, req.Log, req.Telemetry)
+	remoteTaggerInstance, err := newRemoteTagger(req.Params, req.Config, req.Log, req.Telemetry)
 
 	if err != nil {
 		return Provides{}, err
 	}
 
+	// Creates the connection to the remote tagger and starts watching for events.
 	req.Lc.Append(compdef.Hook{OnStart: func(_ context.Context) error {
+		remoteTaggerInstance.telemetryTicker = time.NewTicker(1 * time.Minute)
```

Comment:
I think it would be nice to extract all this to a private start function for readability.
Same for stop.

Reply:
Done 👍